registry/extension/oci/oci.go

```diff
@@ -95,9 +95,28 @@ func (d *ociNamespace) GetRepositoryRoutes() []extension.Route {
 }
 
 // GetRegistryRoutes returns a list of extension routes scoped at a registry level
-// There are no registry scoped routes exposed by this namespace
 func (d *ociNamespace) GetRegistryRoutes() []extension.Route {
-	return nil
+	var routes []extension.Route
+
+	if d.discoverEnabled {
```

Comment:
Will extensions be listed if they are disabled?

Reply:
Discussed offline that due to discover being considered as any other extension, if it isn't specified in the config then the discover route will not reachable. For consistency we'll leave the behavior as such.